pkgs/by-name/sq/squawk-chat/package.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchgit
+, cmake
+, kconf
+, lmdb
+, qxmpp
+, boost
+, libomemo-c
+, pkg-config
+, libsForQt5
+, imagemagick
+
+, kwalletSupport ? false # secure password storage
+, kioSupport ? false # better show in folder action
+, kconfSupport ? false # system themes support
+}:
+
+stdenv.mkDerivation rec {
+  pname = "squawk-chat";
+  version = "0.2.3";
+
+  src = (fetchgit {
+    url = "https://git.macaw.me/blue/squawk";
+    rev = "fb843a13463e6600f86d9dde5aa728cdcfa57259";
+    hash = "sha256-dPeB2DOJ22shgeV6UBXb6Kcswre4q3CRaWogLsLyl2U=";
+  }).overrideAttrs (_: {
+    GIT_CONFIG_COUNT = 1;
+    GIT_CONFIG_KEY_0 = "url.https://git.macaw.me/.insteadOf";
+    GIT_CONFIG_VALUE_0 = "gitea@git.macaw.me:";
+  });
```

Comment:
Maybe can you use `fetchFromGitea` ?
```
src = fetchFromGitea {
    domain = "git.macaw.me/";
    owner = "blue";
    repo = "squawk";
    rev = "v${version}";
    sha256 = "";
  };
  ```
Please use a tag reference whenever possible. If you want to use a specific commit, change the version to follow the conventions.

Reply:
I have not managed to use `fetchFromGitea`, as the upstream repo uses git over ssh for submodules, and `.overrideAttrs` is not supported by `fetchFromGitea`. There may be another way to do this that I am not aware of, though.

I have switched to using tag references

Reply:
Disregard my previous comment, I have found out how to do this with `fetchFromGitea`, I was just missing some parentheses